docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -79,8 +79,9 @@ Block types:
  - ``sonata_mosaic_description``: this block will be always on screen and should represent the entity's name.
 
 
-The ``ObjectMetadata`` object is returned by the related admin class,
-for instance the MediaBundle defines the method as::
+The ``ObjectMetadata`` object is returned by the related admin class, and can be
+used to define which image field from the entity will be displayed if available.
+For instance, the SonataMediaBundle defines the method as::
```

Comment:
Please use .. code-block:: php instead of ::

Reply:
Done.